No Default CRAN

When trying to use `make serve` to preview changes locally, I was receiving the following error:
``` 
Error in contrib.url(repos, "source") :
    trying to use CRAN without setting a mirror
Calls: install_required_packages -> install.packages -> contrib.url
Execution halted
make: *** [install-rmd-deps] Error 1
```

I was able to resolve by setting my default CRAN mirror following a suggestion from @zkamvar by adding `options(repos = c(CRAN = "https://cran.rstudio.org"))` to my `.Rprofile`, but others may be running into similar issues.

--- a/bin/dependencies.R
+++ b/bin/dependencies.R
@@ -9,6 +9,12 @@ install_required_packages <- function(lib = NULL, repos = getOption("repos", def
     c("rprojroot", "desc", "remotes", "renv"),
     rownames(installed.packages(lib.loc = lib))
   )
+  # The default installation of R will have "@CRAN@" as the default repository, which directs contrib.url() to either
+  # force the user to choose a mirror if interactive or fail if not. Since we are not interactve, we need to force the
+  # mirror here.
+  if ("@CRAN@" %in% repos) {
+    repos <- c(CRAN = "https://cran.rstudio.com/")
+  }
 
   install.packages(missing_pkgs, lib = lib, repos = repos)
 

